helpers/mysql/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History: opentelemetry-helpers-mysql
+
+### v0.1.0 / 2023-07-13
+
+* Initial release.

```

Comment:
Is the CHANGELOG automatically generated? I put this here as a boilerplate.

Reply:
Yes the `CHANGELOG` is auto generated by the release scripts in the `.toys` directory